### ISSUE TITLE
Update wechaty-puppet-wechat provider docs page

### ISSUE THREAD

**Issue by huan** [wechaty/docusaurus#738]
According to <https://github.com/wechaty/wechaty-puppet-wechat/issues/127> and <https://wechaty.js.org/2021/04/13/wechaty-uos-web/>, the docs page <https://wechaty.js.org/docs/puppet-providers/wechat> need to be updated.

@gengchen528 would you like to update it for the community, because you are the hero who has enhanced this puppet? :)

**Comment by lijiarui** [comment]
ping @leochen-g 

@huan I think you mentioned the wrong person...

Also, @Rohitesh-Kumar-Jain I think maybe our Tech Writer maybe update this for the community, could you like to help?

**Comment by Rohitesh-Kumar-Jain** [comment]
> Also, @Rohitesh-Kumar-Jain I think maybe our Tech Writer maybe update this for the community, could you like to help?

yes, I think this can be assigned to the Tutorials team as they aren't supposed to do documentation of advanced bots.

**Comment by sbis04** [comment]
But @Rohitesh-Kumar-Jain I think the references team is already working on this page, also they have done a PR #1108

**Comment by Rohitesh-Kumar-Jain** [comment]
> But @Rohitesh-Kumar-Jain I think the references team is already working on this page, also they have done a PR #1108 

Yeah @shraddhavp and @Soumi7 are already working on that 😅
